README.md

```diff
@@ -238,8 +238,9 @@ Plugins which affect the Rollup workflow.
 - [html-entry](https://github.com/leogr/rollup-plugin-html-entry) – Allows use HTML Script Tags as entry points.
 - [incremental](https://github.com/mprt-org/rollup-plugin-incremental) - Recompile only changed modules in watch mode.
 - [jscc](https://github.com/aMarCruz/rollup-plugin-jscc) – Conditional compilation and declaration of ES6 imports.
+- [make](https://github.com/btmorex/rollup-plugin-make) - Build dependency files suitable for make.
```

Comment:
Also, I noticed while doing this that the alphabetical ordering was off, and there was no period at the end of the sentence for `make`. So I fixed both.